tests/test_wentian.py

```
@@ -0,0 +1,857 @@
+                            all_contain_tags = False
+                            break
+                
+                success = structure_ok and all_contain_tags
+                expected_count = 4  # doc1(ml), doc4(python), doc5(ml), doc6(python)
+                count_correct = n_returned == expected_count
+                
+                self._record_collection_op("get-where-in", "get", "$in运算符筛选",
+                    {"where": {"tags": {"$in": ["ml", "python"]}}, "limit": 10},
+                    {"structure_ok": structure_ok, "n_returned": n_returned, "all_contain_tags": all_contain_tags, "count_correct": count_correct},
+                    success and count_correct,
+                    structure if not structure_ok else f"筛选验证失败: 返回{n_returned}个匹配记录，期望{expected_count}个")
```

Ruff (0.14.8)

Undefined name `structure`

(F821)